### PR TITLE
Set up a "Hello world" project as one app with Webpack

### PR DESCRIPTION
**Pull request by shihabrafio**
- Create a new Rails app called 'hello-rails-react'.
-     Use the command rails new -j webpack my_new_rails_app as described in the previous lesson's tutorial.
- Verify that the jsbundling-rails gem is added to your Gemfile.
- Initialize your project with Git.
- Install needed node dependencies as described in the previous lesson's tutorial.
- Create a babel.config.js in the root folder of your project as described in the previous lesson's tutorial.
- Update webpack.config.js to include the babel loader as described in the previous lesson's tutorial.
- Make sure that your project has a Postgres database set up.
- Create a table for storing your messages and create 5 different greetings.
- Create an API endpoint that selects a random greeting from your table (you will need a controller with an action and Rails route).
- Create a static view that will be the root of your app.
- 
- Create the App component with react-router and render it in your static view.
- Create the Greeting component that will display a greeting. Set it up as a route in your App component.
- Create a store, an action, and a reducer that will connect to your API endpoint to get the random greeting.
- Display the random greeting in your Greeting component.

### CODE REVIEW

File: app/javascript/application.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+// Entry point for the build script in your package.json
+// import "@hotwired/turbo-rails"
+// import "./controllers"
```
Comment (jerryowusu) [inline app/javascript/application.js]: Kindly do not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history. :pray:

File: config/routes.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  # # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  namespace :api do
+    resources :greetings, only: :index
+   
+   end
+  #  greetings#index
+   root 'root#index'
+  # # Defines the root path route ("/")
+  # get 'greeting', to: '/api/greetings#index'
+  # # root "articles#index"
```
Comment (jerryowusu) [inline config/routes.rb]: Kindly do not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history. 🙏

**Review by jerryowusu — CHANGES_REQUESTED** [review]
Hi @shihabrafio , 

Good job so far!
Your project is looking fantastic so far :star_struck: 

However, there are a few changes you need to make in order to move on to the next level :sunglasses:

almost got it


## Highlights
Neat and well-organized code :+1: 
Controller set up :+1: 
Added model :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @shihabrafio👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- Good job with the Hello World project. ✔️
- No linter errors. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**